src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -89,6 +89,18 @@ public function __construct(#[\SensitiveParameter] \PDO|string $connOrDsn, strin
         parent::__construct($namespace, $defaultLifetime);
     }
 
+    public static function createConnection(#[\SensitiveParameter] string $dsn, array $options = []): \PDO|string
+    {
+        if ($options['lazy'] ?? false) {
+            return $dsn;
```

Comment:
I think it makes the most sense to just return the DSN when `lazy` is `true` since the adapter itself supports this and creates the connection lazily.

Reply:
this should be the default to me

Reply:
Done